frame/staking/src/offchain_election.rs

```diff
@@ -113,15 +113,15 @@ pub(crate) fn compute_offchain_election<T: Trait>() -> Result<(), OffchainElecti
 	// process and prepare it for submission.
 	let (winners, compact, score) = prepare_submission::<T>(assignments, winners, true)?;
 
-	// defensive-only: active era can never be none except genesis.
-	let era = <Module<T>>::active_era().map(|e| e.index).unwrap_or_default();
+	// defensive-only: current era can never be none except genesis.
+	let current_era = <Module<T>>::current_era().unwrap_or_default();
```

Comment:
Also I don't think this is defensive, if current_era is none it means there is no current_era, maybe the staking module hasn't started yet, but anyway I think it should return OffchainElectionError, but this is out of scope of the PR